README.md

```diff
@@ -46,15 +46,15 @@ It exposes the following metrics:
 | Metric | Description |
 |-|-|
 | `unused_disks_count` | How many unused disks are in this provider |
-| `unused_disks_size_gb` | Total size of unused disks in this provider in GB |
+| `unused_disks_total_size_bytes` | Total size of unused disks in this provider in bytes |
 | `unused_disk_size_bytes` | Size of each disk in bytes |
-| `unused_disks_last_used_at` | Last timestamp (unix ms) when this disk was used. GCP only! |
-| `unused_provider_duration_ms` | How long in milliseconds took to fetch this provider information |
+| `unused_disks_last_used_timestamp_seconds` | Last timestamp (unix seconds) when this disk was used. GCP only! |
+| `unused_provider_duration_seconds` | How long in milliseconds took to fetch this provider information |
```

Comment:
Here the unit is seconds but the description says milliseconds, we should fix this.